railties/lib/rails/application/default_middleware_stack.rb

```diff
@@ -49,6 +49,7 @@ def build_stack
           middleware.use ::Rails::Rack::Logger, config.log_tags
           middleware.use ::ActionDispatch::ShowExceptions, show_exceptions_app
           middleware.use ::ActionDispatch::DebugExceptions, app, config.debug_exception_response_format
+          middleware.use ::ActionDispatch::ActionableExceptions
```

Comment:
Shouldn't this be included only in development by default? Or should we check for smth like `consider_all_requests_local`?

Reply:
Yup, we definitely should check at least `consider_all_requests_local`. Will tweak it today.

Reply:
Haven't seen any tweaking happening 😊